internal/ingress/annotations/disableproxyintercepterrors/main.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
2023 haha

Reply:
In my defense - that PR is really that old.

Fixed.